README.md

```diff
@@ -13,181 +12,243 @@
-- From JuliaCon 2023: A short 15 minute overview of Catalyst as of version 13 is
-available in the talk [Catalyst.jl, Modeling Chemical Reaction Networks](https://www.youtube.com/watch?v=yreW94n98eM&ab_channel=TheJuliaProgrammingLanguage).
-- From JuliaCon 2022: A three hour tutorial workshop overviewing how to use
-  Catalyst and its more advanced features as of version 12.1. [Workshop
-  video](https://youtu.be/tVfxT09AtWQ), [Workshop Pluto.jl
-  Notebooks](https://github.com/SciML/JuliaCon2022_Catalyst_Workshop). 
-- From SIAM CSE 2021: A short 15 minute overview of Catalyst as of version 6 is
-available in the talk [Modeling Biochemical Systems with
-Catalyst.jl](https://www.youtube.com/watch?v=5p1PJE5A5Jw).
-- From JuliaCon 2018: A short 13 minute overview of Catalyst when it was known
-  as DiffEqBiological in older versions is available in the talk [Efficient
-  Modelling of Biochemical Reaction
-  Networks](https://www.youtube.com/watch?v=s1e72k5XD6s)
-
-Finally, an overview of the package and its features (as of version 13) can also be found in its corresponding research paper, [Catalyst: Fast and flexible modeling of reaction networks](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1011530).
+An overview of the package and its features (as of version 13) can also be found in its corresponding research paper, [Catalyst: Fast and flexible modeling of reaction networks](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1011530).
 
 ## Features
 
-- A DSL provides a simple and readable format for manually specifying chemical
-  reactions.
-- Catalyst `ReactionSystem`s provide a symbolic representation of reaction networks,
-  built on [ModelingToolkit.jl](https://docs.sciml.ai/ModelingToolkit/stable/) and
-  [Symbolics.jl](https://docs.sciml.ai/Symbolics/stable/).
-- Non-integer (e.g. `Float64`) stoichiometric coefficients are supported for generating
-  ODE models, and symbolic expressions for stoichiometric coefficients are supported for
-  all system types.
-- The [Catalyst.jl API](http://docs.sciml.ai/Catalyst/stable/api/catalyst_api) provides functionality for extending networks,
-  building networks programmatically, network analysis, and for composing multiple
-  networks together.
-- `ReactionSystem`s generated by the DSL can be converted to a variety of
-  `ModelingToolkit.AbstractSystem`s, including symbolic ODE, SDE and jump process
-  representations.
-- Coupled differential and algebraic constraint equations can be included in
-  Catalyst models, and are incorporated during conversion to ODEs or steady
-  state equations.
-- Conservation laws can be detected and applied to reduce system sizes, and generate
-  non-singular Jacobians, during conversion to ODEs, SDEs, and steady state equations.
+#### Features of Catalyst
+- [The Catalyst DSL](@ref ref) provides a simple and readable format for manually specifying reaction 
```

Comment:
You can't use `@ref` in the README can you?

Reply:
Good point, yeah, will remove these. Haven't actually been able to build the docs yet to check.